myna/application/bnpy/sample.py

```diff
@@ -0,0 +1,73 @@
+import numpy as np
+from .bnpy import bnpy_module_dependency_error_msg
+
+
+def get_representative_distribution(
+    data, n0=1000, dn=1000, conv_abs=1e-3, conv_rel=1e-1, conv_count=3, bins=10
+):
+
+    # Load app-specific dependencies
+    try:
+        import ot
+    except Exception as e:
+        print(e)
+        bnpy_module_dependency_error_msg()
```

Comment:
```suggestion
    except ImportError:
        raise ImportError('bnpy requires "pip install .[bnpy]" optional dependencies!!')
```

Reply:
One important thing is that this will only except import errors (it's pretty unlikely anything else gets through in this case, but it's better to be specific). But I'm not sure it's worth having individual functions either